lib/unsplash/photo.rb

```diff
@@ -57,10 +59,18 @@ def random(categories: nil, collections: nil, featured: nil, user: nil, query: n
           h:        height,
           orientation: orientation
         }.select { |k,v| v }
-
-        photo = Unsplash::Photo.new JSON.parse(connection.get("/photos/random", params).body)
-        photo.user = Unsplash::User.new photo.user
-        photo
+        if count
+          params[:count] = count
+          photos = parse_list connection.get("/photos/random/", params).body
```

Comment:
@razasayed No, it's okay. Like I said above, it's not a performance concern, and I expect to be in here in the future to rework some of the parsing code anyway. So you can leave it like this. 😄